examples/layer-browser/src/map.jsx

```diff
@@ -169,10 +165,6 @@ export default class Map extends PureComponent {
             <ViewportLabel>Orbit View (PlotLayer only, No Navigation)</ViewportLabel>
           </View>
 
-          <div style={NAVIGATION_CONTROL_STYLES}>
-            <NavigationControl />
```

Comment:
I guess it's a bit of a refactor to make the Map the root and bring back support for controls. Note to self to bring this back with a deck navigation widget..